.github/workflows/push.yml

```
@@ -310,3 +310,37 @@ jobs:
             ${{ github.sha }} \
             $(git merge-base ${{ github.event.pull_request.base.sha }} ${{ github.sha }}) \
             > $GITHUB_STEP_SUMMARY
+  WASM32_WASI:
+    name: "WASM32_WASI_${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}"
```

```suggestion
  WASI:
    name: "WASI_${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}"
```

just to make the CI job name shorter/nicer...

---

I would keep `WASM32_WASI` if that's fine, given that WASM64 will eventually come